common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -293,49 +300,53 @@ div.video {
           font-size: em(14);
         }
 
-        li {
+        a {
+          @extend %video-button;
           float: left;
-          margin-bottom: 0;
+          background-image: url('../images/vcr.png');
+          background-position: 15px 15px ;
+          background-repeat: no-repeat;
+          border-left: none;
+          box-shadow: 1px 0 0 #555;
+          padding: 0 lh(.75);
+          width: 14px;
 
-          a {
-            @extend %video-button;
-            background-image: url('../images/vcr.png');
-            background-position: 15px 15px ;
-            background-repeat: no-repeat;
-            border-left: none;
-            box-shadow: 1px 0 0 #555;
-            padding: 0 lh(.75);
-            width: 14px;
-
-            &:focus {
-              @extend %ui-depth4;
-              position: relative;
-              outline: $white dotted thin;
-              outline-offset: -2px;
-            }
+          &:focus {
+            @extend %ui-depth4;
+            position: relative;
+            outline: $white dotted thin;
+            outline-offset: -2px;
+          }
 
-            &:empty {
-              height: 46px;
-              background-position: 15px 15px;
-            }
+          &:empty {
+            height: 46px;
+            background-position: 15px 15px;
+          }
 
-            &.play {
-              background-position: 17px -114px;
-            }
+          &.play {
+            background-position: 17px -114px;
+          }
 
-            &.pause {
-              background-position: 16px -50px;
-            }
+          &.pause {
+            background-position: 16px -50px;
           }
 
-          div.vidtime {
-            font-weight: bold;
-            line-height: 46px; //height of play pause buttons
-            -webkit-font-smoothing: antialiased;
-            padding-left: lh(.75);
-            @media (max-width: 1120px) {
-              padding-left: lh(0.5);
-            }
+          &.skip {
+            background-image: none;
+            text-indent: 0;
+            width: initial;
+            white-space: nowrap;
+          }
+        }
+
+        div.vidtime {
```

Comment:
> NIT: A small note again here on class naming. Could this be "video-duration" (if this is the text for the duration)? This is what I'm guessing it is for based on the current class name.

I don't think that it is a part of this PR. It might require a lot of changes (js, unit tests, bok-choy, etc).